.gitignore

```diff
@@ -0,0 +1,15 @@
+#screenshot & videos
+cypress/screenshots/*
+cypress/videos/*
+
+#reports
+mochawesome-report
+mochawesome.json
+
+#node_moduels
+node_modules
+results
+
+.DS_Store
```

Comment:
Why did you add this ?

Reply:
my PC create this file, we don't need this